etcd/etcd.go

```diff
@@ -1651,8 +1655,8 @@ func (obj *EmbdEtcd) StartServer(newCluster bool, peerURLsMap etcdtypes.URLsMap)
 	cfg := embed.NewConfig()
 	cfg.Name = memberName // hostname
 	cfg.Dir = obj.dataDir
-	cfg.ACUrls = obj.clientURLs
-	cfg.APUrls = peerURLs
+	cfg.ACUrls = obj.advertiseClientURLs
```

Comment:
Want to put these below the other two so that everything is in the same order throughout?
Also, do we want to do an `if len(...) > 0` check before we do the assignment, and if not default to the obj.clientUrls/peerUrls, or does that happen automatically with etcd internally?